test/runtests.jl

```diff
@@ -1,6 +1,7 @@
 using LazySets, LazySets.Approximations, Test, LinearAlgebra, SparseArrays
 import IntervalArithmetic, Expokit, Optim
 using IntervalArithmetic: IntervalBox
+using TaylorModels
```

Comment:
Can you instead add the package under `import`? Then you also do not need to write all these `LazySets.` prefixes. If you need one or two function s from the package, you can get them via `using TaylorModels: fx, fy` as in the line above.

Reply:
Yes, Indeed that is a better way.